BeamEnergy/BeamEnergy.py

```diff
@@ -0,0 +1,60 @@
+""""
+   A facade device for calculating beam's energy at the end of the linac.
```

Comment:
1. This docstring looks like describing the class, not the module. Therefore IMO it should be moved inside the class. At the module level you can have a one-liner saying that there's a device classed called `BeamEnergy` and a `run` method for it.
2. The calculation doesn't have to happen at the end of the linac 😄 
3. This description is really laconic. I'd opt for some more information on how the energy is calculated (it's a 2nd order polynomial) and what the calculation's based on (on a magnet power supply's current).